### ISSUE TITLE
Alignment of the last response box 

### ISSUE THREAD

**Issue by nelsonlapf** [rapidpro/ureport#262]
Hi guys,

We have detected that the alignment of the last response box when the language is Arabic (right to left) is bigger than it should be to the left. 

![syria box](https://cloud.githubusercontent.com/assets/23530401/21983032/4d5f31d2-dbd6-11e6-81dc-46f88441938f.JPG)

And as you can see, for the other languages (left to right), we don´t have this spacing so large.

![uk box](https://cloud.githubusercontent.com/assets/23530401/21982934/bfa7f8a6-dbd5-11e6-964a-f63ef85ca45a.JPG)

We are thinking that it´s only a detail in the CSS code.

**Comment by norkans7** [comment]
@nelsonlapf that Message Us button should not be there. It was added in #203 which is still open since we wait for the design team to come up with a proper design on how to add those button.

The approach is #203 was breaking some sites already

**Comment by teehamaral** [comment]
So if we remove the code of the PR #203 should this problem be resolved?

**Comment by norkans7** [comment]
Good catch @teehamaral It is not resolved. Happy to check a PR for that
